app/src/main/java/org/mifos/selfserviceapp/ui/activities/ClientListActivity.java

```diff
@@ -0,0 +1,130 @@
+package org.mifos.selfserviceapp.ui.activities;
+
+import android.app.ProgressDialog;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Toast;
+
+import org.mifos.selfserviceapp.R;
+import org.mifos.selfserviceapp.data.Client;
+import org.mifos.selfserviceapp.presenters.ClientListPresenter;
+import org.mifos.selfserviceapp.ui.adapters.ClientListAdapter;
+import org.mifos.selfserviceapp.ui.views.ClientListView;
+import org.mifos.selfserviceapp.utils.Constants;
+import org.mifos.selfserviceapp.utils.RecyclerItemClickListener;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * @author Vishwajeet
+ * @since 14/07/16
+ */
+public class ClientListActivity extends BaseActivity implements RecyclerItemClickListener.OnItemClickListener, ClientListView {
```

Comment:
You should consider wrapping your code after a certain number of characters, typically 80 or 100. This makes it a lot more readable.